docs/topics_5/web-adding-global-custom-rules.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='adding_global_custom_rules_{context}']
+= Adding global custom rules
+
+Global rules apply to all projects and are automatically applied to analyses for which they fit the definitions of a project.
+
+The {ProductShortName} includes a preconfigured set of global rules and you can define your own custom global rules.
+
+These rules are displayed and defined on the *Rules configuration* page, which contains two tabs: *System rules* and *Custom rules*. *System rules* contains a preconfigured set of global rules. You can add your own rules in the *Custom rules* tab.
+
+For information on writing custom {ProductShortName} rules, see the {ProductShortName} link:{ProductDocRulesGuideURL}[_{RulesDevBookName}_].
+
+.Rules configuration
+image::web-rules-config_51.png[Rules configuation]
+
+Each row in the tables displayed in *Rules configuration* shows the short path, source, target, and the number of rules in the rule set.
+
+You can upload rule sets or register a server path that contains a rule set.
+
+
+.Procedure
+In the navigation pane, click *Rules configuration*.
+
+. Click *Add rules*.
+. To upload rules:
+
+.. In the *Upload* tab, click *Browse*.
+.. Select a rules file (`\*.mta.xml` or `*.windup.xml`).
+.. Click *Close*.
+
+. To register a server path:
+
+.. Click the *Server path* tab.
+.. Enter the *Rules path* of the server-side file or directory containing the rule files in the field.
+.. Select *Scan recursively* if relevant.
+.. Click *Save*.
+
+. Click *Next*.
++
+[NOTE]
+====
+Uploading labels stores them directly in the {ProductShortName} server.
+
+Registering the directory path of a file tells the {ProductShortName} server the location of the labels. This method is useful if you expect to update the labels because it ensures that the {ProductShortName} server always uses the latest version of the application.
+====
++
+The *Custom rules* table includes the rules.
```

Comment:
includes > contains the new custom rules.

Are you sure it's a table? I think it's called a list in UI context